lib/galaxy/tool_util/data/__init__.py

```diff
@@ -102,9 +103,30 @@ def exists(self, path: str) -> bool:
             return os.path.exists(path)
 
 
+ErrorListT = List[str]
```

Comment:
You can also use `ErrorListT` in the signature of the `extend_data_with()` and `parse_file_fields()` methods.